app/controllers/players_controller.rb

```diff
@@ -0,0 +1,35 @@
+class PlayersController < ApplicationController
+	def index
+		@players = Player.all
+	end
+
+	def new 
+		@player = Player.new
+	end
+
+	def create
+		@player = Player.create(player_params)
+		redirect_to root_path
+	end
+
+	def edit
+		@player = Player.find(params[:id])
+	end
+
+	def update
+		@player = Player.find(params[:id])
+		return redirect_to root_path if @player.update(player_params)
+		redirect_to edit_player_path(params[:id])
+	end
+
+	def destroy
+		@player = Player.find(params[:id])
+		@player.destroy
+		redirect_to players_path
```

Comment:
I know I used `redirect_to :back` in the ideator lesson, but it's actually a bad idea - I need to update the lessons.
Generally you want to use a `_path` instead, so I've changed that here.